strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/ServerConfigurationController.java

```diff
@@ -253,6 +267,53 @@ public ResponseEntity setServerSettings(@RequestBody @Validated ServerSettingsFo
         return getSuccessfulResponseEntity(SUCCESSFUL_SAVE_SERVER_SETTINGS, acceptHeader);
     }
 
+    private void validateServerSettingsForm(ServerSettingsForm form,
+                                            BindingResult bindingResult)
+    {
+        if (form != null)
```

Comment:
I'm not sure what error should I add here...

Reply:
Hmm... Actually I think it would be better if we returned `400 Bad Request`. We could do something like this:

```suggestion
// controller
if (serverSettingsForm == null) {
    return getBadRequestResponseEntity("Empty form was provided", acceptHeader);
}
      
validateServerSettingsForm(serverSettingsForm, bindingResult); // remove the `if` check in the method.
```

Reply:
@steve-todorov: Done!